compiler/src/main/java/com/squareup/anvil/compiler/codegen/Contribution.kt

```diff
@@ -24,7 +24,6 @@ import dagger.Module
 import dagger.Provides
 import dagger.multibindings.IntoMap
 import dagger.multibindings.IntoSet
-import java.util.Comparator
```

Comment:
KtLint didn't catch this because there's a `Comparator` reference at the bottom of the file.  It's actually referencing the built-in `kotlin.Comparator` typealias, but it seems KtLint errs on the side of caution.